gui/src/loader.rs

```diff
@@ -181,13 +181,21 @@ impl Loader {
                 info!("Stopping internal daemon...");
                 daemon.stop();
                 info!("Internal daemon stopped");
-                if self.gui_config.internal_bitcoind_exe_config.is_some() {
-                    if let Some(daemon_config) = daemon.config() {
-                        if let Some(bitcoind_config) = &daemon_config.bitcoind_config {
-                            stop_internal_bitcoind(bitcoind_config);
-                        }
-                    }
+            }
+        }
+        if self.gui_config.internal_bitcoind_exe_config.is_some() {
```

Comment:
Do we need to check that the Liana daemon is not external?

Reply:
Internal daemon and managed bitcoind are an invariant, you are right that adding a check would be good

Reply:
Yes, I think so too. If a user starts their managed bitcoind before opening Liana, then we'll stop it when they close Liana, but it's unlikely they'll do this and so not worth adding extra logic. In any case, I guess this could happen whether the Liana daemon is internal or external. Perhaps we would need a simple flag to indicate if we had started bitcoind.

Reply:
In this case maybe remove the log below?
```rust
info!("Stopped liana managed bitcoind");
```

As this looks like a success message whereas it might have failed.

Reply:
Good idea, addressed in eb10cdacd6ed42c7294f454da62381e3e91269dc